common/locales/de.json

```diff
@@ -103,7 +103,7 @@
         "toggleAsbplayerSubtitleTrack2": "Untertitel 2 in asbplayer umschalten",
         "toggleAsbplayerSubtitleTrack3": "Untertitel 3 in asbplayer umschalten",
         "toggleAsbplayerSubtitleTracks": "Untertitel in absplayer umschalten",
-        "toggleAsbplayerBlurTrack": "Toggle subtitle track {{trackNumber}} blur in asbplayer",
```

Comment:
You should keep the old key `toggleAsbplayerBlurTrack` as well since the same loc file is used for all versions of the extension.

Reply:
reverted the old key